autogpt_platform/backend/backend/blocks/jina_chunking.py

```diff
@@ -0,0 +1,63 @@
+import requests
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from backend.data.model import BlockSecret, SchemaField, SecretField
+
+
+class JinaChunkingBlock(Block):
+    class Input(BlockSchema):
+        texts: list = SchemaField(description="List of texts to chunk")
+        api_key: BlockSecret = SecretField(
+            key="jina_api_key", description="Jina API Key"
+        )
```

Comment:
use the api credentials instead

https://docs.agpt.co/server/new_blocks/#blocks-with-authentication